core/src/main/resources/i18n/displayStrings.properties

```
@@ -2693,6 +2693,13 @@ tradeDetailsWindow.tradeState=Trade state
 tradeDetailsWindow.agentAddresses=Arbitrator/Mediator
 tradeDetailsWindow.detailData=Detail data
 
+txDetailsWindow.headline=Transaction Details
+txDetailsWindow.btc.note=You have sent BTC.
+txDetailsWindow.bsq.note=You have sent BSQ funds.  \
+  Please note that BSQ is colored Bitcoin, it will not show up in a BSQ explorer until at least 1 Bitcoin block has been confirmed.
```

I got the impression this string is saying the BSQ transaction will show in the BSQ explorer as soon as 1 more bitcoin block is confirmed, whether it includes the BSQ transaction or not.

```suggestion
txDetailsWindow.bsq.note=You have sent BSQ funds.  \
  BSQ is colored bitcoin, so the transaction will not show in a BSQ explorer until it has been confirmed in a bitcoin block.
```

---

Thankyou @m52go, change applied.